prototype/Makefile

```diff
@@ -23,7 +23,7 @@ help:
 
```

Comment:
Can the prototype dir be historical only? I would prefer to keep it as a branch / commit but lose it on main merge

Reply:
I merge it to the v1-milestone branch in the other PR for history/reference purposes.

I expect us to have a PR where we just delete it in the future (once main is clean and functioning), and that ways we'll have the historical context via git.